### PR TITLE
Switch to integer sequence numbers so they're wire friendly

### PR DESCRIPTION
**Pull request by Vagabond**
Additionally, I think this fixes a sequence number reuse bug.

### CODE REVIEW

File: src/relcast.erl [hunk 1]
```diff
@@ -366,19 +367,19 @@ take(ForActorID, State = #state{pending_acks = Pending}, _) ->
...
-                            {ok, Ref, Msg,
-                             State#state{last_sent = maps:put(ForActorID, {CF2, Key2}, State#state.last_sent),
+                            {Seq2, State2} = make_seq(ForActorID, State),
+                            Pends1 = Pends ++ [{Seq2, CF2, Key2, Multicast}],
+                            {ok, Seq2, Msg,
+                             State2#state{last_sent = maps:put(ForActorID, {CF2, Key2}, State#state.last_sent),
                                          pending_acks = maps:put(ForActorID, Pends1, Pending)}};
                         not_found ->
-                            {not_found, State#state{last_sent=maps:put(ForActorID, {CF, Key}, State#state.last_sent)}}
+                            {not_found, State#state{last_sent=maps:put(ForActorID, none, State#state.last_sent)}}
```
Comment (madninja) [inline src/relcast.erl]: Are you going to undo this change since it didn't resolve the bug?

**Comment by evanmcc** [comment]
merging as per slack convo
